dicat/lib/dicom_anonymizer_methods.py

```diff
@@ -259,6 +262,11 @@ def pydicom_zapping(dicom_file, dicom_fields):
 
     dicom_dataset = dicom.read_file(dicom_file)
 
+    # tags to force insert in the final file
+    forceInsertTags = [tag for tag in dicom_fields 
+                       if 'ForceInsert' in dicom_fields[tag]
+                       and dicom_fields[tag]['ForceInsert']]
```

Comment:
forceInsertTags = [tag for tag in dicom_fields if dicom_fields[tag].get('ForceInsert')]

Reply:
Yes, but because it is a condition check, it should even be `forceInsertTags = [tag for tag in dicom_fields if dicom_fields[tag].get('ForceInsert', False)]` in the case 'ForceInsert' does not exist